specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+        "Manual"
+      ],
+      "readOnly": true,
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CategoryType",
+        "modelAsString": true
+      }
+    },
+    "ResourceStatus": {
+      "description": "Indicates the resource status.",
+      "enum": [
+        "Healthy",
+        "Unhealthy",
+        "NotApplicable"
+      ],
+      "readOnly": true,
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ResourceStatus",
+        "modelAsString": true
+      }
+    },
+    "Assessment": {
+      "type": "object",
+      "description": "A class represent the assessment.",
+      "properties": {
+        "name": {
+          "description": "The name of the assessment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "severity": {
+          "$ref": "#/definitions/AssessmentSeverity",
+          "readOnly": true,
+          "description": "The severity level of this assessment."
+        },
+        "description": {
+          "description": "The description of the assessment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "remediation": {
+          "description": "The remediation of the assessment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isPass": {
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "readOnly": true,
+          "description": "Indicates whether all the resource(s) are compliant.",
+          "x-ms-enum": {
+            "name": "IsPass",
+            "modelAsString": true
+          }
+        },
+        "policyId": {
+          "description": "The policy id mapping to this assessment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceList": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of resource assessments.",
+          "items": {
+            "description": "Single resource metadata.",
+            "$ref": "#/definitions/AssessmentResource"
+          },
+          "x-ms-identifiers": [
+            "resourceId"
+          ]
+        }
+      }
+    },
+    "AssessmentResource": {
+      "type": "object",
+      "description": "A class represent the assessment resource.",
+      "properties": {
+        "resourceId": {
+          "description": "The Id of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceStatus": {
+          "description": "Resource status.",
+          "readOnly": true,
+          "$ref": "#/definitions/ResourceStatus"
+        },
+        "reason": {
+          "description": "The reason for the N/A resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "statusChangeDate": {
+          "description": "The status change date for the resource. For unavailable date, set it as N/A.",
+          "type": "string",
```

Comment:
Would this be date-time format? What does 'set it as N/A' mean when its readonly, do you mean the server will be returning the string 'N/A'? I wonder if that makes it easier or harder to parse than just returning null/undefined when its unknown.